package.nls.json

```
@@ -40,5 +40,6 @@
     "containerApps.disconnectRepo": "Disconnect from Repo",
     "containerApps.openGitHubRepo": "Open Repo in GitHub",
     "containerApps.startStreamingLogs": "Start Streaming Logs...",
-    "containerApps.stopStreamingLogs": "Stop Streaming Logs..."
+    "containerApps.stopStreamingLogs": "Stop Streaming Logs...",
+    "containerApps.createAcr": "Create Azure Container Registry"
```

Add the ellipsis since I'm assuming it involves user input.

```suggestion
    "containerApps.createAcr": "Create Azure Container Registry..."
```